bus/rabbitmq/bus.js

```diff
@@ -77,6 +77,15 @@ function RabbitMQBus (options) {
       done();
     });
 
+    if (options.enableConfirms) {
+      self.connection.createConfirmChannel().then(function (channel) {
+        channel.on('error', channelError);
+        self.confirmChannel = channel;
```

Comment:
Can you set prefetch on this channel similar to: https://github.com/mateodelnorte/servicebus/blob/master/bus/rabbitmq/bus.js#L70-L72 (this is new)